storage/storagedriver/overlayfs-driver.md

```diff
@@ -490,14 +442,6 @@ Both `overlay2` and `overlay` drivers are more performant than `aufs` and
   far larger latencies if searching through many AUFS layers. `overlay2` supports
   multiple layers as well, but mitigates any performance hit with caching.
 
-- **Inode limits**. Use of the `overlay` storage driver can cause excessive
```

Comment:
@thaJeztah Since we're deprecating the overlay driver, why should we keep documentation on the older driver? If you want to have something on why we are deprecating it, I'm for it.

@andrewhsu What are your thoughts?

Reply:
We should have that info because even if the driver is deprecated, there are still people using it (and will be, for at least a few years). Having a doc to help them out is good, especially since we already have that info written -- it just need to be moved into a separate doc having a preamble/disclaimer like "Note overlay is deprecated and should not be used; the information below is kept here mostly for historical purposes" or so.